### DISCUSSION TITLE
lacking customizability

### DISCUSSION THREAD

**Discussion by ChannyClaus** [community/community#28426] [Other Feature Feedback, Questions, & Ideas]
i've blocked some users that i would like to avoid seeing but for you page keeps shoving them into my face. i would certainly appreciate it if you guys can either not make this page to be the default landing page or allow the users to customize it further in a way that respects the list of blocked users.

**Comment by jfullstackdev** [comment]
this is a know bug and the Staff is already working on this, will wait for the update
